fetch.bs

```diff
@@ -4002,13 +4010,20 @@ steps:
 
  <li><p>Let <var>timingInfo</var> be <var>response</var>'s <a for=response>timing info</a>.
 
+ <li><p>Let <var>cacheState</var> be <var>response</var>'s <a for=response>cache state</a>.
+
  <li><p>If <var>timingInfo</var> is null, then return.
 
- <li><p>If <var>response</var>'s <a for=response>timing allow passed flag</a> is not set, then set
- <var>timingInfo</var> to a new <a for=/>fetch timing info</a> whose
- <a for="fetch timing info">start time</a> and
- <a for="fetch timing info">post-redirect start time</a> are <var>timingInfo</var>'s
- <a for="fetch timing info">start time</a>.
+ <li><p>If <var>response</var>'s <a for=response>timing allow passed flag</a> is not set, then
```

Comment:
As this `<li>` contains multiple flow elements the `<p>` needs to indented on a newline.

Reply:
ack